packages/x-charts/src/ChartsTooltip/ChartsTooltipContainer.tsx

```diff
@@ -93,9 +93,11 @@ function ChartsTooltipContainer(inProps: ChartsTooltipContainerProps) {
       popperRef.current?.update();
     };
 
+    element.addEventListener('pointerdown', handleMove);
```

Comment:
The position wasn't being set on pointerdown, which looked odd in mobile when pressing, the tooltip would appear at the border of the screen, then jump to touch.

Reply:
Use custom tooltip: https://stackblitz.com/edit/x7sypmzs-nreczuaz?file=src%2Findex.tsx
Gotta be using touch

Reply:
> Also, we should probably rename handleMove to handlePointerEvent or similar.

I'm wondering if the pointerDown should be a bit more special. If the tooltip is closed, it open it. If the tooltip is already open, it does nothing.

Reply:
> Use custom tooltip: [stackblitz.com/edit/x7sypmzs-nreczuaz?file=src%2Findex.tsx](https://stackblitz.com/edit/x7sypmzs-nreczuaz?file=src%2Findex.tsx) Gotta be using touch

Thanks! Managed to reproduce it, it seems the tooltip renders in the last position until you move the pointer.